app/app.go

```diff
@@ -250,11 +253,12 @@ type PstakeApp struct {
 	LiquidStakeIBCKeeper  liquidstakeibckeeper.Keeper
 
 	// make scoped keepers public for test purposes
-	ScopedIBCKeeper           capabilitykeeper.ScopedKeeper
-	ScopedTransferKeeper      capabilitykeeper.ScopedKeeper
-	ScopedICAHostKeeper       capabilitykeeper.ScopedKeeper
-	ScopedICAControllerKeeper capabilitykeeper.ScopedKeeper
-	ScopedLSCosmosKeeper      capabilitykeeper.ScopedKeeper
+	ScopedIBCKeeper            capabilitykeeper.ScopedKeeper
+	ScopedTransferKeeper       capabilitykeeper.ScopedKeeper
+	ScopedICAHostKeeper        capabilitykeeper.ScopedKeeper
+	ScopedICAControllerKeeper  capabilitykeeper.ScopedKeeper
+	ScopedLSCosmosKeeper       capabilitykeeper.ScopedKeeper
+	ScopedLiquidStakeIBCKeeper capabilitykeeper.ScopedKeeper
```

Comment:
https://github.com/cosmos/ibc-go/blob/v6.0.0/docs/migrations/v5-to-v6.md#controller-apis
> Application developers that wish to consume IBC packet callbacks and react upon packet acknowledgements must continue using the controller submodule's legacy APIs. The authentication modules will not need a ScopedKeeper anymore, though, because the channel capability will be claimed by the controller submodule.